<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskItem</class>
 <widget class="QWidget" name="TaskItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>77</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="pushButton_Delete">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::EditDelete"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_Title">
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_DeadLine">
     <property name="text">
      <string>DeadLine</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_Priority">
     <property name="text">
      <string>Priority</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_Complete">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_Edit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::MailMessageNew"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
